Signal/src/ViewControllers/AttachmentApprovalViewController.swift

```diff
@@ -140,10 +147,11 @@ class AttachmentApprovalViewController: UIViewController, OWSAudioAttachmentPlay
         audioPlayButton.autoSetDimension(.height, toSize:buttonSize)
         subviews.append(audioPlayButton)
 
-        if let fileExtensionLabel = createFileExtensionLabel() {
+        if let fileNameLabel = createFileNameLabel() {
+            subviews.append(fileNameLabel)
+        } else if let fileExtensionLabel = createFileExtensionLabel() {
             subviews.append(fileExtensionLabel)
         }
```

Comment:
nit: Shouldn't happen with well behaving clients, but otherwise I think we will break the layout

```
else {   
  subviews.append(unknownFilenameLabel)
}
```